.github/workflows/tests.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [3.7, 3.8, 3.9]
+        python-version: [3.7, 3.8, 3.9, 3.10]
```

Comment:
`3.10` should be `'3.10'` as per https://github.com/actions/setup-python/issues/160